src/formio/components/npFamilyMembers.ts

```diff
@@ -0,0 +1,21 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type NpFamilyMembersInputSchema = InputComponentSchema<
+  Record<string, boolean>,
```

Comment:
what happens if we use `never` for the value type? Could we use this to signal that this component will never have a value, as the component type is not going to be supported directly in the form renderer? It is effectively a builder-only component.